executor/prepared.go

```diff
@@ -336,7 +336,12 @@ func ResetStmtCtx(ctx context.Context, s ast.StmtNode) {
 	sc.TimeZone = sessVars.GetTimeZone()
 
 	switch stmt := s.(type) {
-	case *ast.UpdateStmt, *ast.DeleteStmt:
+	case *ast.UpdateStmt:
+		sc.IgnoreTruncate = false
+		sc.OverflowAsWarning = false
+		sc.TruncateAsWarning = !sessVars.StrictSQLMode || stmt.IgnoreErr
+		sc.InUpdateOrDeleteStmt = true
+	case *ast.DeleteStmt:
```

Comment:
Delete has `IGNORE` option too.
How about support it as well?

Reply:
Support delete ignore with another pr would be better